### DISCUSSION TITLE
Finding Cost-Effective Paths in AI Automation

### DISCUSSION THREAD

**Discussion by HMB14** [community/community#147284] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

As someone deeply passionate about AI automation, I've conducted preliminary research online and discovered that Zapier and MAKE emerge as the leading platforms in this field. Between the two, MAKE stands out for its competitive pricing, user-friendly interface, and versatile functionality. However, a significant concern arises from these platforms' reliance on various paid third-party services, such as Airtable, which require separate subscriptions. This dependency creates a cascade of monthly fees across multiple services. I'm seeking cost-effective alternatives to these platforms and wondering if Python could be a viable solution. Though my Python programming skills are currently limited, I'm willing to invest time in developing this expertise if it offers a more sustainable approach.

**Comment by Logophoman** [comment]
Go and Look at Langchain actions and tools 😊 ChatGPT can help you out with your python implementations - it's important that you learn to read and understand the code not so much write it anymore... Yes I'd probably do it myself if I were you

**Reply by HMB14** [reply]
Thank you.
